nncf/onnx/graph/model_transformer.py

```diff
@@ -397,11 +407,12 @@ def _apply_qdq_node_removing_transformations(
         :param transformations: Nodes removing transformations.
         :return: Model with removed nodes.
         """
-        onnx_graph = ONNXGraph(model)
         for transformation in transformations:
-            node = onnx_graph.get_node_by_name(transformation.target_point.target_node_name)
+            node_mapping = get_node_mapping(model)
+            edge_node_mapping = get_edge_node_mapping(model)
+            node = node_mapping[transformation.target_point.target_node_name]
 
-            node_children = onnx_graph.get_children(node)
+            node_children = get_children(node, edge_node_mapping)
```

Comment:
Really, how is the new approach better than the previous one? The raw line count has increased in this particular case, and barely changed if we look at the line count of the PR as a whole. 

You are now exposing the implementation details (numerous `mapping`s) while the user doesn't really need to know about this - instead the user needs to get children of a node with a given name from a graph, which is exactly what the previous implementation was about, and the code itself was readable and consistent with the purpose. Now the code is cluttered with extra calls and dicts that aren't used per se but are necessary to fit to the new approach.